packages/tools/src/tools/annotation/DragProbeTool.ts

```diff
@@ -51,7 +51,7 @@ class DragProbeTool extends ProbeTool {
   }
 
   postMouseDownCallback = (
```

Comment:
Needs a `postTouchDownCallback` exposed to have touch events dispatched to this tool.

Reply:
"postTouchDownCallback" or "postTouchStartCallback"?

I believe we're using the "postTouchStartCallback", see below:

packages/tools/src/eventDispatchers/touchEventHandler/touchstart.ts

```
if (activeTool && typeof activeTool.postTouchStartCallback === 'function') {
    const consumedEvent = activeTool.postTouchStartCallback(evt);

    if (consumedEvent) {
      // If the tool claims it consumed the event, prevent further checks.
      return;
    }
  }
```

Reply:
should be postTouchStart callback you are correct. Rather than copying the mouse function you can just call the postMouseDownCallback function inside of postTouchStart callback